### PR TITLE
update to fonk-final-form adaptor

### PR DESCRIPTION
**Pull request by brauliodiez**

### CODE REVIEW

File: form-libraries/react-final-form.mdx [hunk 1]
```diff
@@ -10,7 +10,59 @@ route: /react-final-form
...
+npm install @lemoncode/fonk @lemoncode/fonk-final-form --save
+```
+
+The only difference when using **fonk-final-form** is: isntead of invoking the _createFomValidation_ method
+to create a validation schema, you will need to invoke _createFinalFormValidation_, example:
+
+```diff
+import { isNumber } from '@lemoncode/fonk-is-number-validator';
+import { minNumberValidator } from './custom-validators';
+- import { createFormValidation, Validators } from '@lemoncode/fonk';
```
Comment (Nasdan) [inline form-libraries/react-final-form.mdx]:
```
- import { createFormValidation, Validators } from '@lemoncode/fonk';
+ import { Validators } from '@lemoncode/fonk';
+ import { createFinalFormValidation } from '@lemoncode/fonk-final-form';
```

File: form-libraries/react-final-form.mdx [hunk 2]
```diff
@@ -10,7 +10,59 @@ route: /react-final-form
...
+      },
+    ],
+  },
+};
+
+- export const formValidation = createFormValidation(validationSchema);
++ export const formValidation = createFinalFormValidation(validationSchema);
+```
+
+Under the hood, what the adaptar does is just return _null_ when a validation succeeds (that's the result
```
Comment (Nasdan) [inline form-libraries/react-final-form.mdx]: Typo `adaptor`

File: form-libraries/react-final-form.mdx [hunk 3]
```diff
@@ -10,7 +10,59 @@ route: /react-final-form
...
 Adding **Fonk** to **React Final Form** allows you to seamlessly add validation capabilities to **React Final Form**.
 
-In this page you will find a set of examples showing wide range of use cases:
+In order to use **Fonk** with **React Final Form** you will have to install **fonk-final-form** adaptor:
+
+```bash
+npm install @lemoncode/fonk @lemoncode/fonk-final-form --save
+```
+
+The only difference when using **fonk-final-form** is: isntead of invoking the _createFomValidation_ method
```
[the remark this answers is not in the document]
Comment (brauliodiez) [reply form-libraries/react-final-form.mdx]: fixed

File: form-libraries/react-final-form.mdx (hunk 1 above)
Comment (brauliodiez) [reply form-libraries/react-final-form.mdx]: fixed

File: form-libraries/react-final-form.mdx (hunk 2 above)
Comment (brauliodiez) [reply form-libraries/react-final-form.mdx]: fixed

**Pull request merged**